### PR TITLE
CVSB-3911+CVSB-3912

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/pipes/format-vrm/format-vrm.pipe.ts [hunk 1]
```diff
@@ -9,6 +9,6 @@ export class FormatVrmPipe implements PipeTransform {
   }
 
   transform(vrm: string): string {
-    return (vrm.length > 4 ? `${vrm.slice(0, 4)} ${vrm.slice(4)}` : vrm);
+    return (vrm && vrm.length > 4 ? `${vrm.slice(0, 4)} ${vrm.slice(4)}` : vrm);
```
Comment (agoncearenco) [inline src/pipes/format-vrm/format-vrm.pipe.ts]: So basically if you try to transform an empty string this method is supposed to return the empty string?

File: src/pages/testing/test-creation/test-create/test-create.html [hunk 2]
```diff
@@ -102,7 +104,7 @@ <h3 class="test-type">{{ vehicleTest.testTypeCategoryName }}</h3>
 
     <ion-card class="card" *ngIf="displayAddVehicleButton">
       <button class="btn-add-test" ion-item detail-none (click)="addTrailer(visitService.visit.tests)">
-        <span>Add a trailer</span>
+        <span>Add a <span *ngIf="!doesHgvExist">vehicle</span><span *ngIf="doesHgvExist">trailer</span></span>
```
[the remark this answers is not in the document]
Comment (agoncearenco) [reply src/pages/testing/test-creation/test-create/test-create.html]: As discussed with @adpetrescu , I applied this change.
